src/views/api-docs-view/server/get-api-docs-static-props/utils/build-version-switcher-data.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiDocsVersionData } from '../../../types'
+import { findLatestStableVersion } from '../../../utils'
+
+/**
+ * Construct a label and options for an API docs version switcher.
+ */
+function buildVersionSwitcherData({
```

Comment:
We translate version data into version switcher options as part of `getStaticProps`, to avoid doing this client-side.